recipes-nymea/nymea-plugins-zigbee/nymea-plugins-zigbee_git.bb

```diff
@@ -0,0 +1,34 @@
+DESCRIPTION = "nymea-plugins-zigbee"
+
+LICENSE = "LGPL-3.0 | NYMEA-COMMERCIAL"
+LIC_FILES_CHKSUM="file://LICENSE.LGPL3;md5=3000208d539ec061b899bce1d9ce9404"
+
+SRC_URI="git://github.com/nymea/nymea-plugins-zigbee.git;protocol=https;branch=master"
+# Release: 1.0.0
+SRCREV="75314cf373476ee20a1545b834d4c66acdb6d851"
+PV = "git${SRCPV}"
+
+DEPENDS += "nymead nymead-native"
+
+inherit qmake5
+
+S = "${WORKDIR}/git"
+
+EXTRA_QMAKEVARS_PRE += "CONFIG+=selection ${PACKAGECONFIG_CONFARGS}"
+
+# One can find all available plugins by running oe-pkgdata-util list-pkgs nymea-plugins after having bitbake'd nymea-plugins
+PACKAGESPLITFUNCS_prepend = " split_nymea_plugins_packages "
+
+python split_nymea_plugins_packages() {
+    nymea_libdir = d.expand('${libdir}/nymea/plugins/')
+    plugins = do_split_packages(d, nymea_libdir, r'^libnymea_integrationplugin(.*)\.so\.*', 'nymea-plugin-%s', 'Nymea integration plugin for %s', extra_depends='')
```

Comment:
could add zigbee outside of the matching pattern and have "Nymea Zigbee integration for %s" as text. But that's nitpick, it's probably just fine like this.

Reply:
For this I decided to keep this as it is for now.